### PR TITLE
add sawyers needed to search filters

### PR DESCRIPTION
**Pull request by cguardia**
This works, but not sure if sawyers will go under certifications or roster requests.

### CODE REVIEW

**Comment by davisagli** [comment]
I think it belongs here under certifications.

File: src/vms/static/app/search.js [hunk 1]
```diff
@@ -306,8 +309,15 @@ var WorkPartyFilters = React.createClass({
 								                showAll={true}
 										updateQuery={this.props.updateQuery} />
 									<hr />
+                                    <MultiChoiceFacet
+                                            name="cert_requests"
+                                            label="Certification Requests"
```
Comment (davisagli) [inline src/vms/static/app/search.js]: label should be "Certifications Required"

Also this facet should only show up if the user has a permission level of Assistant Crew Leader, Crew Leader, or Admin

File: src/vms/static/app/search.js [hunk 2]
```diff
@@ -219,6 +219,9 @@ var WEEKDAY_OPTIONS = [
 	'Weekdays',
 ]
 
+var CERT_REQUESTS = [
+    "Sawyers Needed"
```
Comment (davisagli) [inline src/vms/static/app/search.js]: I think just list it as "Sawyer" here. "Needed" is redundant with "required" in the heading for the list.

File: src/vms/static/app/search.js (hunk 1 above)
Comment (cguardia) [inline src/vms/static/app/search.js]: @davisagli the show_all option prefixes the label with "All ", so it reads "All Certifications Required". I think that's a bit confusing. Maybe just "All Certifications"?

Comment (davisagli) [inline src/vms/static/app/search.js]: I think I would just set show_all to false for this facet.

**Comment by cguardia** [comment]
@davisagli ok, I think it's ready.

**Pull request merged**